### PR TITLE
Prepare 18.0.0

### PR DESCRIPTION
**Pull request by ybiquitous**
> Is there anything in the PR that needs further explanation?

- Write changelog
- Bump dependencies
- Try a [merge queue](https://docs.github.com/en/repositories/configuring-branches-and-merges-in-your-repository/configuring-pull-request-merges/managing-a-merge-queue).

### CODE REVIEW

File: .github/workflows/nodejs.yml [hunk 1]
```diff
@@ -6,8 +6,7 @@ on:
       - main
       - 'dependabot/**'
   pull_request:
-    branches:
-      - '**'
+  merge_group:
```
Comment (ybiquitous) [inline .github/workflows/nodejs.yml:9]: [note] Try a [merge queue](https://docs.github.com/en/repositories/configuring-branches-and-merges-in-your-repository/configuring-pull-request-merges/managing-a-merge-queue).

**Review by jeddy3 — APPROVED** [review]
LGTM, thank you!

Let's add to the merge queue 😄

**Comment by ybiquitous** [comment]
Thanks for the review. I've fixed the changelog a bit via cb8705d. I'll merge and release it now!

**Comment by ybiquitous** [comment]
[18.0.0](https://github.com/stylelint/eslint-config-stylelint/releases/tag/18.0.0) is out! 🎉 
See also https://www.npmjs.com/package/eslint-config-stylelint

**Pull request merged**